seL4-platforms/platforms.yml

```diff
@@ -171,7 +171,7 @@ platforms:
     aarch_hyp: [64]
     platform: imx93
     march: armv8a
-    no_hw_test: true
+    req: imx93a
```

Comment:
Apologies, I should have noticed the missing `-DNUM_NODES=2` earlier, but I guess this is the place to add it.

Reply:
Right, yes, that should go in here, I think.